### PR TITLE
Add new logo files

### PR DESCRIPTION
**Pull request by matthiasgeiger**
Still [WIP]:
- [x] Add PNG versions in Java (different abstractions for different sizes)
- [x] Create new `*.ico` and `*.icns` files
- [x] Check installer
- [x] Issue: Invisible Icon in Win 10 due to same background color (see also comment from Tobias)
- [ ] create file icon for associated `*.bib` files
  - create ico files for Win and Mac
  - configure install4j

Normal todos:
- [x] Change in CHANGELOG.md described
- ~~[ ] Tests created for changes~~
- [ ] Screenshots added (for bigger UI changes)
- [x] Manually tested changed features in running JabRef
- ~~[ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)~~
- ~~[ ] If you changed the localization: Did you run `gradle localizationUpdate`?~~

### CODE REVIEW

**Comment by tobiasdiez** [comment]
Most of the apps on the Windows Start Screen (Win 10) have a white foreground and a colorful background. Can we also create an inverted image of the new logo to adhere to this style? (Or after skimming through the [guidelines](https://msdn.microsoft.com/windows/uwp/controls-and-patterns/tiles-and-notifications-app-assets) I think a white Icon with transparent background is needed)

**Comment by tobiasdiez** [comment]
Most of the apps on the Windows Start Screen (Win 10) have a white foreground and a colorful background. Can we also create an inverted image of the new logo to adhere to this style? (Or after skimming through the [guidelines](https://msdn.microsoft.com/windows/uwp/controls-and-patterns/tiles-and-notifications-app-assets) I think a white Icon with transparent background is needed)

**Comment by tobiasdiez** [comment]
Most of the apps on the Windows Start Screen (Win 10) have a white foreground and a colorful background. Can we also create an inverted image of the new logo to adhere to this style? (Or after skimming through the [guidelines](https://msdn.microsoft.com/windows/uwp/controls-and-patterns/tiles-and-notifications-app-assets) I think a white Icon with transparent background is needed)

**Comment by tobiasdiez** [comment]
Most of the apps on the Windows Start Screen (Win 10) have a white foreground and a colorful background. Can we also create an inverted image of the new logo to adhere to this style? (Or after skimming through the [guidelines](https://msdn.microsoft.com/windows/uwp/controls-and-patterns/tiles-and-notifications-app-assets) I think a white Icon with transparent background is needed)

**Comment by lenhard** [comment]
What is the status here? It would be cool to have the new icon soon :-)

**Comment by matthiasgeiger** [comment]
Basically it's ready to merge - further improvement would be to create a file-icon but this is just a nice-to-have feature.

As I don't have that much time in the next weeks I'll merge the current status (after rebasing and fixing the resulting errors due to the package changes :wink:)

**Comment by matthiasgeiger** [comment]
Okay... should be ready-for-review from my side..

@chochreiner would be cool if you could check whether the installation on a Mac is successfull and showing the new logo. Thanks!

**Comment by chochreiner** [comment]
works fine 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 ## [Unreleased]
 
 ### Changed
+- JabRef has a new logo!
```
Comment (stefan-kolb) [inline CHANGELOG.md]: Maybe add the name and link to the designer?

**Pull request merged**